src/qt/guiutil.cpp

```diff
@@ -119,6 +120,14 @@ void setupAddressWidget(QValidatedLineEdit *widget, QWidget *parent)
     widget->setCheckValidator(new BitcoinAddressCheckValidator(parent));
 }
 
+void setButtonShortcuts(QPushButton* button, const QList<QKeySequence>& shortcuts)
```

Comment:
This function could be more generic if it would accept `QAbstractButton*` instead.